modules/extensions-spring-support/src/test/resources/dynamic-config-expiration.xml

```diff
@@ -140,4 +166,42 @@
         <logger level="ERROR" message="#[ import * from dw::Runtime --- 'Waiting for 10000ms' wait 10000]"/>
     </flow>
 
+    <flow name="dynamicWithShortExpirationForPagedOperation">
+        <heisenberg:failing-connectivity-paged-operation failOn="#[vars.failOn]" failAtClose="false" config-ref="heisenbergWithShortExpiration"/>
+        <test:processor class="org.mule.test.module.extension.DynamicConfigExpirationTestCase$CaptureProcessor"/>
+        <foreach collection="#[payload]">
+            <logger level="ERROR" message="#[payload]" />
+            <test:processor class="org.mule.test.module.extension.DynamicConfigExpirationTestCase$CaptureProcessor"/>
+            <logger level="ERROR" message="#[%dw 2.0
```

Comment:
this is unnecesary. The test:processor operation has a waitTime parameter which automatically sleeps

Reply:
yes but waitTime is not applied when you use a custom processor due to how FunctionalTestProcessor is coded, nor can you access the waitTime variable in the custom processor :/
I can replace the dataweave sleeps with <test:processor waitTime="1000"> but I think it is too much overhead maybe?

Reply:
on the contrary... using DW for sleeping is the overkill thing